cdntaxreceipts.functions.inc

```diff
@@ -618,7 +610,7 @@ function cdntaxreceipts_configure_inkind_fields() {
     $group = array(
       'title' => 'In-kind donation fields',
       'extends' => array( 'Contribution' ),
-      'extends_entity_column_value' => array( $contrib_type_id ),
+      'extends_entity_column_value' => [$financialTypeID],
```

Comment:
What fields are supposed to go into this group, and why is that not part of this PR? On test2, when I create a contribution with fund of In-kind, then this field set opens and shows

DESCRIPTION OF PROPERTY * |  
APPRAISED BY * |  
ADDRESS OF APPRAISER * |  
ORIGINAL COST

Reply:
@JoeMurray the custom fields are declared just few lines below at [line 626 ](https://github.com/JMAConsulting/CDNTaxReceipts/pull/16/files#diff-2ce320da8fdb5162b9d06cb4e2ed8259R626) which is a part of the old code here - https://github.com/jake-mw/CDNTaxReceipts/blob/master/cdntaxreceipts.functions.inc#L613